pages/users_test.py

```diff
@@ -16,30 +16,52 @@
 
 import flask
 import html5lib
+import os
+
+from google.cloud import ndb
+from pathlib import Path
+from unittest import mock
 
 from internals import user_models
 from pages import users
 
 test_app = flask.Flask(__name__)
 
 
+# Load testdata to be used across all of the CustomTestCases
+TESTDATA = testing_config.Testdata(
+  os.path.abspath(os.path.dirname(__file__)),
```

Comment:
Nit: This repeated code could probably be simpler it it only passed `__file__` and the Testdata constructor did all the OS path stuff.